homeassistant/components/venstar/__init__.py

```diff
@@ -89,5 +89,5 @@ def device_info(self) -> DeviceInfo:
             name=self._client.name,
             manufacturer="Venstar",
             model=f"{self._client.model}-{self._client.get_type()}",
-            sw_version="{}.{}".format(*(self._client.get_firmware_ver())),
+            sw_version=f"{self._client.get_firmware_ver()[0]}.{self._client.get_firmware_ver()[1]}",
```

Comment:
`self._client.get_firmware_ver()` seems like an expensive operation, calling it twice changes the way the code works.

Reply:
It is not an expensive operation. Just reads an internal property.